images-list

```diff
@@ -303,6 +307,8 @@ k8s.gcr.io/sig-storage/livenessprobe rancher/mirrored-sig-storage-livenessprobe
 kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 0.1.151
 kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 1.1.0
 kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 0.1.151
+lachlanevenson/k8s-kubectl rancher/mirrored-lachlanevenson-k8s-kubectl v1.22.2
+library/bash rancher/mirrored-library-bash 5.1.4
```

Comment:
We should just use `rancher/shell` for this or `rancher/kubectl` or use the default SLES image here. Why do we specifically need to mirror another bash image?

Reply:
See https://github.com/rancher/image-mirror/pull/239#discussion_r886803508

Reply:
Yep. Done. Commit 7722deff7ffc1605eae25b4ecbd45a7cf42480a4